spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binder.java

```diff
@@ -32,21 +32,21 @@
 public interface Binder<T> {
 
 	/**
-	 * Bind a message consumer on a channel
+	 * Bind the target component as a message consumer to the logical entity identified by the name.
```

Comment:
Sorry guys, maybe I was too quick on this. If you come up with a better phrasing, just confirm it here, and I can add an additional commit